dawn/test/unit-test/dawn/CodeGen/samples/copy_stencil.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+##===-----------------------------------------------------------------------------*- Python -*-===##
+##                          _
+##                         | |
+##                       __| | __ ___      ___ ___
+##                      / _` |/ _` \ \ /\ / / '_  |
+##                     | (_| | (_| |\ V  V /| | | |
+##                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+##
+##
+##  This file is distributed under the MIT License (MIT).
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+
+import os
+
+import dawn4py
+from dawn4py.serialization import SIR
+from dawn4py.serialization import utils as sir_utils
+from google.protobuf.json_format import MessageToJson
+
+stencil_name = "copy_stencil"
+output_file = f"{stencil_name}.cpp"
+
+if __name__ == "__main__":
+    interval = sir_utils.make_interval(SIR.Interval.Start, SIR.Interval.End, 0, 0)
```

Comment:
Is there a reason why the main code is both setup and then compilation? Its would intuitively look nicer if we have a function that sets the SIR and than have the main just call 
```
setup()
compile()
rename()
```
Thoughts?

Reply:
I see the benefit of separating the setup part from the rest. However i would keep these 2 lines together
```
dawn4py.compile(sir, backend="cuda", serialize_iir=True, output_file=output_file)
os.rename(stencil_name + ".0.iir", stencil_name + ".iir")
```
as the "module" which would make most sense to me here is `sir_to_iir()`, but I have no way to specify to dawn4py that I don't want the `.0` extension.